assets/src/edit-story/app/media/utils/useProcessVideo.js

```diff
@@ -123,7 +123,7 @@ function useProcessVideo({
           additionalData: { alt: oldResource.alt, title: oldResource.title },
         });
       };
-      process();
+      return process();
```

Comment:
added a return here so that i can use `.finally` to set the optimizing button back to default state.